### DISCUSSION TITLE
Change commands made by Vaillant sensoCOMFORT (VRC 7XX) don't appear ebusd log file

### DISCUSSION THREAD

**Discussion by VivLloyd** [john30/ebusd#1406] [General]
If I send a message to ebusd to say change the Set-back temperature (z1NightTemp) to 21 via MQTT, I will see a corresponding log entry e.g. "2024-12-03 10:22:03.476 [mqtt notice] write basv z1NightTemp: 21". If I do the same thing via my Vaillant sensoCOMFORT VCR 720f/2 I see no log entry. Why is this? Is there a way to log all commands sent via Vaillant control devices?

**Comment by sebska** [comment]
I think you are sending your commands to the control unit, so the control unit is the master for your heating. If you ar doing it via eBUS it is in the logs. The sensoCOMFORT does not send any messages to somebody if you change it directly within (he is the Master!). You need to read alle the values in intervals an compare if somthing has changed within the "master". There is no way to tell the sensoCOMFORT to sennd all changes made within to send it to the eBUS. There is no need for sensoCOMFORT to do this.

**Reply by VivLloyd** [reply]
Thanks for your response. I came to the same conclusion after further consideration. The only visible ebus messages are those that need to be transmitted across the bus to other slave devices to initiate a change or read a value. Filtering messages that are sent to each slave would enable a map to be created of messages served by each slave.
